cli/pcluster/config_sanity.py

```diff
@@ -532,47 +533,84 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
             ]:
                 self.__fail(resource_type, "Region %s is not supported with batch scheduler" % self.region)
 
+            # Check spot bid percentage
+            if "SpotPrice" in resource_value:
+                spot_price = int(resource_value["SpotPrice"])
+                if spot_price > 100 or spot_price < 0:
+                    self.__fail(resource_type, "Spot bid percentage needs to be between 0 and 100")
+
+            # Check sanity on desired, min and max vcpus
+            if (
+                "DesiredSize" not in resource_value
+                or "MinSize" not in resource_value
+                or "MaxSize" not in resource_value
+            ):
+                # this should never occur
```

Comment:
If this never occurs I would just remove this if

Reply:
Done